core/models.py

```diff
@@ -11,10 +18,20 @@ def __str__(self):
 
 class SearchQuery(models.Model):
 
+    """
+    Thia model is used to store the search queries in the database.
+    Attributes:
+        query (str): The search query received in API calls (can be null).
+        curriculum (str): The curriculum of the search query (can be null).
+        It is a foreign key to the Curriculum model.
+        education_level (str): The education level of the search query (can be null).
+        It is a foreign key to the EducationLevel model.
+    """
+
     class Meta:
         verbose_name_plural = "Search Queries"
     
-    query = models.CharField(max_length=255, verbose_name="query")
+    query = models.CharField(max_length=255, verbose_name="query", default=None, null=True, blank=True)
     curriculum = models.CharField(max_length=255, verbose_name="curriculum", default=None, null=True, blank=True)
     education_level = models.CharField(max_length=255, verbose_name="education_level", default=None, null=True, blank=True)
```

Comment:
Same, is mentioned in the docstring that the education level field is a foreign key to the Curriculum Model but it is defined as a CharField. Please fix either the Field or the docstring, accordingly.

Reply:
Resolved in the previous conversation.